src/pages/instances/Events.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, useEffect, useState } from "react";
+import { LxdEvent } from "types/event";
+import { useEventQueue } from "context/eventQueue";
+
+const Events: FC = () => {
+  const eventQueue = useEventQueue();
+  const [eventWs, setEventWs] = useState<WebSocket | null>(null);
+
+  const handleEvent = (event: LxdEvent) => {
+    const entry = eventQueue.get(event.metadata.id);
```

Comment:
It took me some time (and a check to the relevant part of the code) to understand/remember what the `entry` was - and I had just checked the code in `eventQueue` shortly before. If one is not aware of the structure of an item in the `eventQueue`, they will have an even harder time. I propose renaming this to something that makes it immediately clear what the entry is about, maybe `eventCallbacks`?